apis/user-java/helm/values.yaml

```diff
@@ -21,17 +21,17 @@ ingress:
     endpoint:
       host: changeme
       paths:
-      - path: /api/user-profile
-        serviceName: api-user-profile-svc
+      - path: /api/user-java
+        serviceName: api-user-java-svc
         servicePort: 80
-      - path: /api/healthcheck/user-profile
-        serviceName: api-user-profile-svc
+      - path: /api/healthcheck/user-java
+        serviceName: api-user-java-svc
         servicePort: 80
-      - path: /api/docs/user-profile
-        serviceName: api-user-profile-svc
+      - path: /api/docs/user-java
```

Comment:
Please fix the docs route so that `/api/docs/user-java` actually works with your code.  Currently the docs are only served up on `/api/swagger-ui.html`